packages/venia-concept/src/actions/checkout/asyncActions.js

```diff
@@ -244,9 +243,10 @@ export const submitOrder = () =>
             );
 
             dispatch(
-                checkoutReceiptActions.setOrderInformation(
-                    getOrderInformation(getState(), response)
-                )
+                checkoutReceiptActions.setOrderInformation({
+                    id: response,
+                    shipping_address
+                })
```

Comment:
We have all the information we need to make this call in scope here, no need to go to the store to get it.

Reply:
Importantly, this has changed to `shipping_address` instead of `billing_address` because that is where `firstname`, `lastname`, and `email` live.

If the user is creating an account it does make more sense to seed the Create Account form with the first name, last name, and email address from their billing address (as opposed to the shipping address, which might be different). 

Because the billing address does not contain this information, and the user can change the values on the Create Account form if they so desire, I have created a new ticket to address the desired design: #1154. I considered it out of scope for this bug ticket.

Reply:
I agree with the creation of the ticket to add those additional fields, but I don't see why you had to change the receipt to pull values from `shipping_address` in this scope. `email, firstname, lastname, telephone` are copied from shipping to billing, so the values are there, and it'll be one less thing they need to do in scope of #1154.

I quickly tested and didn't run into a problem. Was there a use case that necessitated that change?

Reply:
Good points, I'll play with this.